.gitignore

```diff
@@ -1,6 +1,9 @@
+# files and folder generated by webgenr
 _website/
+templates
```

Comment:
we need templates directory at the root, since those files are what are accessed by RustEmbed and used to auto-populate a new project

Reply:
good catch.  We might want to add another .gitignore to the `examples` folder.

Reply:
removing `templates` from `.gitignore`